src/components/Navigation/NavLogo.js

```diff
@@ -27,21 +27,29 @@ const PageNameEl = styled.span`
 `;
 const NavDividerEl = styled(DividerEl)`
   height: 3rem;
-  background-color: ${({ theme }) => theme.border};
+  background-color: ${PALETTE.white80};
```

Comment:
Is this new design? No more visible divider line?

Reply:
Hmm it was looking invisible to me beforehand. Lemme double check this, I may have committed too quickly.

Reply:
I think you are right (I asked Charles just in case since this wasn't brought up when we had in person QA).

Reply:
I changed this to `white60`, `white80` is the same as the background. Looks like `theme.border` is no more, which would explain why it's not showing up at https://developers.stellar.org/docs/ right now